src/coreclr/jit/hwintrinsicarm64.cpp

```diff
@@ -221,7 +355,7 @@ bool HWIntrinsicInfo::isScalarIsa(CORINFO_InstructionSet isa)
 //    pImmUpperBound [OUT] - The upper incl. bound for a value of the intrinsic immediate operand
 //
 void HWIntrinsicInfo::lookupImmBounds(
-    NamedIntrinsic intrinsic, int simdSize, var_types baseType, int* pImmLowerBound, int* pImmUpperBound)
+    NamedIntrinsic intrinsic, int simdSize, var_types baseType, int immNumber, int* pImmLowerBound, int* pImmUpperBound)
```

Comment:
Why is this method taking `immNumber`? It doesn't look like its being used anywhere...

Reply:
That's because we don't yet have any intrinsics with two immediates. Next PR will add:

```
            case NI_Sve_SaturatingDecrementBy16BitElementCount:
            case NI_Sve_SaturatingDecrementBy32BitElementCount:
..etc...
            case NI_Sve_SaturatingIncrementBy64BitElementCountVector:
            case NI_Sve_SaturatingIncrementBy8BitElementCountVector:
                if (immNumber == 1)
                {
                    immLowerBound = 1;
                    immUpperBound = 16;
                }
                else
                {
                    assert(immNumber == 2);
                    immLowerBound = (int)SVE_PATTERN_POW2;
                    immUpperBound = (int)SVE_PATTERN_ALL;
                }
                break;
```

Reply:
worth adding the comment about the new argument in method docs. ok to do it in follow-up PR when you add cases to handle `NI_Sve_SaturatingDecrementBy*`